presto-memory/src/test/java/io/prestosql/plugin/memory/TestMemorySmoke.java

```diff
@@ -106,7 +105,7 @@ public void testJoinDynamicFilteringNone()
                 "SELECT * FROM lineitem JOIN orders ON lineitem.orderkey = orders.orderkey AND orders.totalprice < 0",
                 withBroadcastJoin(),
                 0,
-                ImmutableSet.of(0, ORDERS_COUNT));
+                ImmutableList.of(0, ORDERS_COUNT));
```

Comment:
nit: we probably could use vararg here

Reply:
Sounds good, changed `assertDynamicFiltering` to accept `int... expectedOperatorRowsRead`.